src/Relation.js

```diff
@@ -156,18 +156,15 @@ utils.addHiddenPropsToTarget(Relation.prototype, {
     const localField = this.localField
     records.forEach((record) => {
       const relatedData = utils.get(record, localField)
-      this.unlinkInverseRecords(relatedData)
+      /* CAN DEPRECATE? */
+      // this.unlinkInverseRecords(relatedData, record)
```

Comment:
This is code-reduplicative and incorrect for many cases, optimally there should only be 1 standard correct method called for all pathways which need to unlink.